src/components/scrollbar/scrollbar_horizontal.ts

```diff
@@ -54,3 +54,7 @@ export class HorizontalScrollBar extends Component<Props, SpreadsheetChildEnv> {
     });
   }
 }
+
+HorizontalScrollBar.props = {
+  position: { type: Object, optional: true },
```

Comment:
This props name is confusing. I feel liks it's a bit strange to have a position that's optional. And it's optional because that's not really the scrollbar position, but an offset to add to the viewportRect. 
Tl;dr : the props should be names offset, not position